lib/chef/provisioning/fog_driver/providers/google.rb

```diff
@@ -35,6 +35,13 @@ def bootstrap_options_for(action_handler, machine_spec, machine_options)
             bootstrap_options[:zone_name] ||= 'europe-west1-b'
             bootstrap_options[:name] ||= machine_spec.name
             bootstrap_options[:disk_size] ||= 10
+            disk_type_prefix = "https://www.googleapis.com/compute/v1/projects/#{compute_options["google_project"]}/zones/#{bootstrap_options[:zone_name]}/diskTypes/"
```

Comment:
It might be interpreted by fog with both version, but here we should get it the way we setted it earlier. iirc i setted the options using the string, so that's why i get it back as a string, did you set it in your config using symbol?